app/scripts/router.js

```diff
@@ -139,7 +141,7 @@ function (
       // back is enabled after the first view is rendered or
       // if the user is re-starts the app.
       this.canGoBack = this.window.sessionStorage.canGoBack || false;
-
+      this.environment = new Environment(this.window);
```

Comment:
i added the option thing in. Did u mean leave out '_' for all the variables and add it in a later commit?